.github/workflows/CI.yml

```diff
@@ -84,3 +84,18 @@ jobs:
             ${{ runner.os }}-node-
       - run: npm ci
       - run: npm run build
+
+  Storybook:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+      - run: npm run test:playground:smoketest
```

Comment:
I'm not sure what you mean 🧐

Reply:
Maybe I know what you mean, yea the intention is that this would carry over to a project that uses the app-web template as a base

Reply:
Awesome, that makes sense to me. I only ask because we do have some tests that only run on the base repo—we run against multiple operating systems, for instance.